lib/stimpack/integrations/factory_bot.rb

```diff
@@ -5,8 +5,10 @@ module Integrations
     class FactoryBot
       def initialize(app)
         return unless app.config.respond_to?(:factory_bot)
+        Stimpack.configure_packs
 
         Packs.all.each do |pack|
+          next if pack.relative_path.glob('*.gemspec').any?
```

Comment:
We don't want to load factories for packs that are gems.

Later, we could consider a first-class API for packs, so this line could be:
```ruby
next if pack.gem?
```